gateway/src/api/latest.rs

```diff
@@ -104,6 +104,23 @@ async fn get_project(
     Ok(AxumJson(response))
 }
 
+async fn get_projects_list(
+    State(RouterState { service, .. }): State<RouterState>,
+    _: User,
```

Comment:
Regarding the account comment, you should be able to get the account name from this guard.

Reply:
@chesedo got it, thank you!